scripts/benchmark.js

```diff
@@ -64,24 +67,82 @@ function test(name, callback) {
   const gzipped = zlib.gzipSync(result);
 
   const parseStart = Date.now();
-  // disable for now we have a failing test in dce
   new Function(result);
   const parseEnd = Date.now();
   const parseNow = parseEnd - parseStart;
 
+  const runTimes = [run];
+
+  for (var i = 1; i < NUM_TEST_RUNS; i++) {
+    const start = Date.now();
+    const result = callback(code);
+    runTimes.push(Date.now() - start);
+  }
+
+  const totalTime = runTimes.reduce((a, b) => a + b, 0);
+  const average = parseInt(totalTime / runTimes.length, 10);
+
   results.push({
     name: name,
     raw: result.length,
     gzip: gzipped.length,
     parse: parseNow,
-    run: run,
+    run: average,
   });
 }
 
-test("babili", function (code) {
+test("babili (best size)", function (code) {
   return babel.transform(code, {
     sourceType: "script",
-    presets: [require("../packages/babel-preset-babili")],
+    minified: true,
+    plugins: [
+      require("babel-plugin-minify-constant-folding"),
+      require("babel-plugin-minify-dead-code-elimination"),
+      require("babel-plugin-minify-flip-comparisons"),
+      require("babel-plugin-minify-guarded-expressions"),
+      require("babel-plugin-minify-infinity"),
+      require("babel-plugin-minify-mangle-names"),
+      require("babel-plugin-minify-replace"),
+      [
+        require("../packages/babel-plugin-minify-simplify/lib/index.js"),
+        { multiPass: true }
+      ],
+      require("babel-plugin-minify-type-constructors"),
+      require("babel-plugin-transform-member-expression-literals"),
+      require("babel-plugin-transform-merge-sibling-variables"),
+      require("babel-plugin-transform-minify-booleans"),
+      require("babel-plugin-transform-property-literals"),
+      require("babel-plugin-transform-simplify-comparison-operators"),
+      require("babel-plugin-transform-undefined-to-void"),
+    ],
+    comments: false,
```

Comment:
Not too happy with this duplication but as options grow, it will diverge more.

Or should we create 2 proper separate presets?